api/v1/deneb/blindedbeaconblockbody.go

```diff
@@ -36,7 +36,7 @@ type BlindedBeaconBlockBody struct {
 	SyncAggregate          *altair.SyncAggregate
 	ExecutionPayloadHeader *deneb.ExecutionPayloadHeader
 	BLSToExecutionChanges  []*capella.SignedBLSToExecutionChange `ssz-max:"16"`
-	BlobKzgCommitments     []deneb.KzgCommitment                 `ssz-max:"4096" ssz-size:"?,48"`
+	BlobKzgCommitments     []deneb.KzgCommitment                 `ssz-max:"4"  ssz-size:"?,48"`
```

Comment:
The latest consensus specs have set max blobs per block to 6, please could you update accordingly throughout the PR?